docs/guides/tutorials/getting_started/initialize_a_data_context.rst

```diff
@@ -24,64 +24,59 @@ or git, you may want to check out the :ref:`supporting_resources` section before
 
     pip install great_expectations
 
-To install from a git branch, use the following command (replace ``develop`` below with the name of the branch you want to use):
 
-.. code-block:: bash
-
-    git clone https://github.com/great-expectations/great_expectations.git
-    cd great_expectations/
-    git checkout develop
-    pip install -e .
-
-To install from a git fork, use the following command (replace ``great-expectations`` below with the name of the fork, which is usually your github username):
-
-.. code-block:: bash
+If you intend to develop within Great Expectations (e.g. to contribute back to the project), and would like to install it from a git branch or a fork, check out :ref:`contributing_setting_up_your_dev_environment` in the contributor documentation.
 
-    pip install -e .
-    git clone https://github.com/great-expectations/great_expectations.git
-    pip install great_expectations/
+Preparation instructions
+---------------------
 
-If you intend to develop within the Great Expectations (e.g. to contribute back to the project), check out :ref:`contributing_setting_up_your_dev_environment` in the contributor documentation.
+For this tutorial, we will use a simplified version of the NYC taxi ride data. We prepared a Docker image that contains a local Postgres database with the data pre-loaded, so you can easily get up and running without any local dependencies.
 
-Download example data
----------------------
+To avoid confusion during the tutorial, we recommend you follow these steps:
 
-For this tutorial, we will use a simplified version of the National Provider Identifier (NPI) database. It's a public dataset released by the `Centers of Medicare and Medicaid Services <https://www.cms.gov/Regulations-and-Guidance/Administrative-Simplification/NationalProvIdentStand/DataDissemination>`_, intended as an authoritative list of health care providers in the United States. NPI data is famously messy---a great place to see the value of data testing and documentation in action.
+#. Make sure you have `Docker<https://www.docker.com/>`_ installed
```

Comment:
Broken link (probably just needs a space before the '<')

![image](https://user-images.githubusercontent.com/332575/90848757-a3dd8200-e33b-11ea-812d-a9b9b187de37.png)